example/eblob_backend.c

```diff
@@ -1078,6 +1080,10 @@ static int blob_send(struct eblob_backend_config *cfg, void *state, struct dnet_
 		if (err)
 			goto err_out_send_fail_reply;
 
+		/* stop if an error is occurred during write */
+		if (ctl->write_error)
+			goto err_out_put;
+
```

Comment:
As I understand this is needed to earlier stop of the server-send iteration over provided keys if there is write error, is that correct?